.eslintrc.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
         "@typescript-eslint/require-await": "off",
         "@typescript-eslint/restrict-template-expressions": "off",
         "@typescript-eslint/unbound-method": "off",
+        "arrow-body-style": "off",
```

Comment:
I think this is a not necessarily helpful, and am getting tired of the linter yelling at me every time I needed to change a function from being implicitly returned to being in block body when extra logic is needed